### PR TITLE
fix(slider): add persistentThumb and valueToLabel props

### LINKED ISSUE #4074: [slider] Option to always show current value (like in versions prior to 9)
### Background

The slider in BaseWeb up to version 9 used to look [like this](https://v8-18-0.baseweb.design/components/slider/):

Screen Shot 2021-02-13 at 12 11 11 PM

Notice how the currently-selected value (60.0) is visible at all times. 

[Since version 9](https://baseweb.design/components/slider/), the slider thumb no longer shows the currently-selected value, unless you hover:
Screen Shot 2021-02-13 at 12 13 19 PM

### Feature request

Add an option to always show the selected value. For example:

```
<Slider
      ...
      showThumbValue={true}
    />
```

or maybe something like:

```
    <Slider
      ...
      overrides={{
        ThumbValue: () => ({
           alwaysVisible: true,
        }),
      }}
    />
```

### PR DESCRIPTION
**Pull request by williamernest**
Fixes #4074 
#### Description

This adds 2 new props: persistentThumb and valueToLabel. 

persistentThumb causes the thumbs/value labels to always display, as they did in prior versions. 
valuetoLabel is a function that's useful to transform the range value to a user-friendly string (see `src/slider/__tests__/slider-custom-label.scenario.js` for an example). 

#### Scope
Minor: New Feature

### CODE REVIEW

File: src/slider/styled-components.js [hunk 1]
```diff
@@ -135,24 +135,29 @@ export const Thumb = styled<StylePropsT>('div', props => {
...
-export const ThumbValue = styled<{}>('div', props => {
-  const {$theme} = props;
+export const ThumbValue = styled<StylePropsT>('div', props => {
+  const {$disabled, $theme} = props;
   return {
     position: 'absolute',
     top: `-${$theme.sizing.scale1400}`,
     ...$theme.typography.font200,
-    backgroundColor: $theme.colors.backgroundInversePrimary,
+    // backgroundColor: $theme.colors.backgroundInversePrimary,
```
Comment (chasestarr) [inline src/slider/styled-components.js:157]:
```suggestion
```

**Review by chasestarr — APPROVED** [review]
Awesome, looking good

**Comment by ericsoco** [comment]
Nice, thanks for jumping on this @williamernest !
Any idea when you'll be cutting a release?

**Comment by williamernest** [comment]
Should go out next week with v10.

**Pull request merged**